examples/GLES31/computeHistogramOfValues_version1.shadertrap

```diff
@@ -0,0 +1,78 @@
+#Copyright 2021 Mostafa Ashraf
+#
+#Licensed under the Apache License, Version 2.0(the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#https: // www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+GL 4.5
+
+CREATE_BUFFER histogram
```

Comment:
I would call this something like `data_values`, because it's not actually the histogram - it's the data we are doing to compute the histogram from.

I also propose that you declare a duplicate buffer, called e.g. `original_data_values`. See below for why.